webdriver_wharf/app.py

```diff
@@ -172,18 +183,21 @@ def container_info(container):
     host = requesting_host()
     host_noport = host.split(':')[0]
 
-    return {
+    container_info = {
         'image_id': container.image_id,
         'checked_out': container.checked_out,
         'checkin_url': 'http://%s/checkin/%s' % (host, container.name),
         'renew_url': 'http://%s/renew/%s' % (host, container.name),
         'webdriver_port': container.webdriver_port,
         'webdriver_url': 'http://%s:%d/wd/hub' % (host_noport, container.webdriver_port),
-        'vnc_port': container.vnc_port,
-        'vnc_display': 'vnc://%s:%d' % (host_noport, container.vnc_port - 5900),
-        'http_port': container.http_port,
-        'fileviewer_url': 'http://%s:%d/' % (host_noport, container.http_port),
     }
+    if vnc_enabled:
+        container_info['vnc_port'] = container.vnc_port
+        container_info['vnc_display'] = 'vnc://%s:%d' % (host_noport, container.vnc_port - 5900)
```

Comment:
I think what I said in the first comment is still good:

>  they should be present in the container info, even if they're None

The docs say that you will get a port and url from those values, but I think I'm painted into a corner on that one, since I love and want this change. :)

Probably rewording the docs to indicate that the values might be None (`null` in JSON) would be nice, and I'll double check that the CFME QE automation will be just fine with the change.